client/mysqlbinlog.cc

```diff
@@ -142,6 +142,7 @@ static char *charset= 0;
 static uint verbose= 0;
 
 static ulonglong start_position, stop_position;
+static const longlong stop_position_default= (longlong)(~(my_off_t)0);
```

Comment:
Why `longlong`? Seems like it should be `ulonglong` like the original type.

Reply:
I thought the same and tried it, but the compiler complains about type narrowing in the argument parsing.  We tell the arg parser that it is a ULL type which should be sufficient.  We could explore that further; my factoring preserves the original behavior.